nifi-assembly/NOTICE

```diff
@@ -818,6 +818,15 @@ The following binary components are provided under the Apache Software License v
        The code for the t-digest was originally authored by Ted Dunning
        A number of small but very helpful changes have been contributed by Adrien Grand (https://github.com/jpountz)
 
+    (ASLv2) subethasmtp
+       The following NOTICE information applies:
+
+       Copyright (C) 2006-2007 SubEthaMail.org
```

Comment:
@joewitt, in regards to your comments on the NOTICE, I can't seem to find this NOTICE information in the version of the dependency that is being used[1]

[1] https://github.com/voodoodyne/subethasmtp/tree/3.1.7